test/persistent_workers.jl

```diff
@@ -0,0 +1,44 @@
+include("testhelpers/PersistentWorkers.jl")
+using .PersistentWorkers
+using Test
+using Random
+using DistributedNext
+
+@testset "PersistentWorkers.jl" begin
+    cookie = randstring(16)
+    port = rand(9128:9999) # TODO: make sure port is available?
```

Comment:
You could open a temporary server with `Sockets.listenany()` and get the port from that.